src/main/config/log4j2.yaml

```diff
@@ -0,0 +1,30 @@
+---
+Configuration:
+  status: warn
+  name: Euphrates
+  properties:
+    property:
+      name: euphratesLog
+      value: euphrates.log
+  thresholdFilter:
+    level: debug
+  appenders:
+    File:
+      name: File
+      fileName: ${sys:euphratesLog}
+      PatternLayout:
+        Pattern: "[%-5p] %d %c - %m%n"
+      Filters:
+        ThresholdFilter:
+          level: debug
+  Loggers:
+    logger:
+      - name: com.patreon.euphrates
+        level: debug
```

Comment:
It's probably not a great idea to have debug logging enabled all the time (or if important logs are logged at debug, they should be logged at info or warn).  In general debug logging is for development or - well - debugging.  If it requires code fixes we can clean up later.

Reply:
Agreed. Cut https://patreon.atlassian.net/browse/INFRA-258 for this.